devito/yask/operator.py

```diff
@@ -28,8 +28,7 @@ class Operator(OperatorRunnable):
     A special :class:`OperatorCore` to JIT-compile and run operators through YASK.
     """
 
-    _default_headers = OperatorRunnable._default_headers
-    _default_headers += ['#define restrict __restrict']
+    _default_headers = OperatorRunnable._default_headers + ['#define restrict __restrict']
```

Comment:
I doubt you want to modify something in yask, perhaps this is just a leftover?

Reply:
I was thinking of making it to follow the pattern of the line bellow.
Maybe that kind of modifications should be left for another PR?

Reply:
ahhh! no you did right, fine ! my bad :)

Reply:
Ok! :)